pkg/pkg.mk

```diff
@@ -21,8 +21,8 @@ $(PKG_BUILDDIR)/.git-downloaded:
 
 clean::
 	@test -d $(PKG_BUILDDIR) && { \
-		git -C $(PKG_BUILDDIR) clean -f ; \
-		git -C $(PKG_BUILDDIR) checkout "$(PKG_VERSION)"; \
+		cd $(PKG_BUILDDIR) && git clean -f ; \
+		cd $(PKG_BUILDDIR) && git checkout "$(PKG_VERSION)"; \
 		$(GIT_APPLY_PATCHES) ; \
 		touch $(PKG_BUILDDIR)/.git-downloaded ; \
 	} > /dev/null 2>&1 || true
```

Comment:
Considering the complexity of the preceding code block, this `|| true` has some notion of "to big to fail". ;-)